paddle/fluid/framework/fleet/fleet_wrapper.cc

```diff
@@ -315,7 +323,14 @@ void FleetWrapper::PushSparseVarsWithLabelAsync(
     }
     int len = tensor->numel();
     int64_t* ids = tensor->data<int64_t>();
-
+    push_values->resize(fea_keys.size() + 1);
+    for (auto& t : *push_values) {
+      t.resize(emb_dim + offset + slot_offset);
+    }
```

Comment:
resize here should be moved out of the loop，because it has nothing to with this loop

Reply:
done